packages/web-components/examples/codesandbox/components/masthead-l1/src/index.js

```diff
@@ -62,4 +56,4 @@ const l1Data = {
   ],
 };
 
-document.querySelector('cds-masthead-composite').l1Data = l1Data;
+document.querySelector('c4d-masthead-container').l1Data = l1Data;
```

Comment:
I'm running into what I think is a Parcel issue. This selector query somehow returns `null`, despite it definitely being in the HTML document.

```
Uncaught TypeError: Cannot set properties of null (setting 'l1Data')
    at parcelRequire.src/index.js.@carbon/ibmdotcom-web-components/es/components/masthead (index.js:59:1)
    at newRequire (ipcinfoCookie.js:32:1)
    at ipcinfoCookie.js:63:1
    at ipcinfoCookie.js:97:10
```

Reply:
Sometimes it's the basics that trip you up. The scripts simply need to be deferred.